### PR TITLE
Mac OS Build

### PR DESCRIPTION
**Pull request by LorenzMeier**
Please cross-test on Linux. This builds on Mac OS (10.10).

### CODE REVIEW

File: src/gazebo_mavlink_interface.cpp [hunk 1]
```diff
@@ -260,14 +263,14 @@ void GazeboMavlinkInterface::ImuCallback(ImuPtr& imu_message) {
     mavlink_hil_sensor_t sensor_msg;
     sensor_msg.time_usec = world_->GetSimTime().nsec*1000;
     sensor_msg.xacc = imu_message->linear_acceleration().x();
-    sensor_msg.yacc = imu_message->linear_acceleration().y();
-    sensor_msg.zacc = imu_message->linear_acceleration().z();
+    sensor_msg.yacc = -imu_message->linear_acceleration().y();
```
Comment (devbharat) [inline src/gazebo_mavlink_interface.cpp]: @tumbili Do we want to keep these changes here or have the IMU rotated in QGC ? I prefer the second one.

File: src/gazebo_mavlink_interface.cpp [hunk 2]
```diff
@@ -260,14 +263,14 @@ void GazeboMavlinkInterface::ImuCallback(ImuPtr& imu_message) {
...
+    sensor_msg.zacc = -imu_message->linear_acceleration().z();
     sensor_msg.xgyro = imu_message->angular_velocity().x();
-    sensor_msg.ygyro = imu_message->angular_velocity().y();
-    sensor_msg.zgyro = imu_message->angular_velocity().z();
+    sensor_msg.ygyro = -imu_message->angular_velocity().y();
+    sensor_msg.zgyro = -imu_message->angular_velocity().z();
     sensor_msg.xmag = mag_I.x;
-    sensor_msg.ymag = mag_I.y;
-    sensor_msg.zmag = mag_I.z;
+    sensor_msg.ymag = -mag_I.y;
```
Comment (devbharat) [inline src/gazebo_mavlink_interface.cpp]: I'd also like if someone had a 2nd look at the units etc of the sensors (pressure etc) being sent from here against what PX4 expects.

**Comment by devbharat** [comment]
Tested on linux, works as expected!

Waiting for the response to the 'convert to NED coordinate system' commit.

File: src/gazebo_mavlink_interface.cpp (hunk 1 above)
Comment (tumbili) [inline src/gazebo_mavlink_interface.cpp]: @devbharat I agree.

**Comment by tumbili** [comment]
@devbharat can you kick out my commits?

**Comment by tumbili** [comment]
@devbharat you could just cherry-pick the one commit

**Comment by devbharat** [comment]
Ok

**Pull request merged**